### ISSUE TITLE
IllegalArgumentException: Claim #2850622277 in C6KLDT6QG: type="Ping", id="com/zerocracy/stk/pm/staff/elections/elect_perfo...

### ISSUE THREAD

**Issue by yegor256** [zerocracy/farm#297]
https://sentry.io/zerocracy/farm/issues/358260610/

```
ClosedChannelException: None
    at sun.nio.ch.FileChannelImpl.ensureOpen(FileChannelImpl.java:110)
    at sun.nio.ch.FileChannelImpl.read(FileChannelImpl.java:147)
    at sun.nio.ch.ChannelInputStream.read(ChannelInputStream.java:65)
    at sun.nio.ch.ChannelInputStream.read(ChannelInputStream.java:109)
    at sun.nio.ch.ChannelInputStream.read(ChannelInputStream.java:103)
...
(45 additional frame(s) were not displayed)

SdkClientException: Unable to execute HTTP request: null
    at com.amazonaws.http.AmazonHttpClient$RequestExecutor.handleRetryableException(AmazonHttpClient.java:1113)
    at com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeHelper(AmazonHttpClient.java:1063)
    at com.amazonaws.http.AmazonHttpClient$RequestExecutor.doExecute(AmazonHttpClient.java:743)
    at com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeWithTimer(AmazonHttpClient.java:717)
    at com.amazonaws.http.AmazonHttpClient$RequestExecutor.execute(AmazonHttpClient.java:699)
...
(14 additional frame(s) were not displayed)

InvocationTargetException: None
    at sun.reflect.GeneratedMethodAccessor595.invoke
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:498)
    at com.zerocracy.farm.reactive.StkGroovy.process(StkGroovy.java:134)
    at com.zerocracy.farm.StkSafe.process(StkSafe.java:86)
...
(24 additional frame(s) were not displayed)

IllegalStateException: java.lang.reflect.InvocationTargetException in com/zerocracy/stk/pm/staff/elections/elect_performer.groovy
    at com.zerocracy.farm.reactive.StkGroovy.process(StkGroovy.java:144)
    at com.zerocracy.farm.StkSafe.process(StkSafe.java:86)
    at com.zerocracy.farm.reactive.brigade.StkWithStatus.process(StkWithStatus.java:55)
    at com.zerocracy.farm.reactive.Brigade.process(Brigade.java:101)
    at com.zerocracy.farm.reactive.Brigade.lambda$process$0(Brigade.java:83)
...
(21 additional frame(s) were not displayed)

IllegalArgumentException: Claim #2850622277 in C6KLDT6QG: type="Ping", id="com/zerocracy/stk/pm/staff/elections/elect_performer.groovy"
    at com.zerocracy.farm.StkSafe.process(StkSafe.java:137)
    at com.zerocracy.farm.reactive.brigade.StkWithStatus.process(StkWithStatus.java:55)
    at com.zerocracy.farm.reactive.Brigade.process(Brigade.java:101)
    at com.zerocracy.farm.reactive.Brigade.lambda$process$0(Brigade.java:83)
    at org.cactoos.func.IoCheckedFunc.apply(IoCheckedFunc.java:66)
...
(20 additional frame(s) were not displayed)

Claim #2850622277 in C6KLDT6QG: type="Ping", id="com/zerocracy/stk/pm/staff/elections/elect_performer.groovy"
```

**Comment by g4s8** [comment]
@0crat waiting #456

**Comment by 0crat** [comment]
> @0crat waiting #456  ([here](https://github.com/zerocracy/farm/issues/297#issuecomment-360091773))

@g4s8 The impediment for [#297](https://github.com/zerocracy/farm/issues/297) was registered successfully by @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by g4s8** [comment]
@yegor256 #456 was merged and some todos were posted, can we close?

**Comment by yegor256** [comment]
@g4s8 thanks

**Comment by 0crat** [comment]
Order was successfully finished: +30 points just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8), total is [+3565](https://www.0crat.com/u/g4s8)

**Comment by 0crat** [comment]
The job [#297](https://github.com/zerocracy/farm/issues/297) is now out of scope

**Comment by 0pdd** [comment]
@yegor256 the puzzle [#963](https://github.com/zerocracy/farm/issues/963) is still not solved; solved: [#476](https://github.com/zerocracy/farm/issues/476), [#477](https://github.com/zerocracy/farm/issues/477).

**Comment by 0pdd** [comment]
@yegor256 all 3 puzzles are solved here: [#476](https://github.com/zerocracy/farm/issues/476), [#477](https://github.com/zerocracy/farm/issues/477), [#963](https://github.com/zerocracy/farm/issues/963).
